### PR TITLE
Implemented a better way to add documents in the Getting Started code example

### LINKED ISSUE #67: A better way to add documents in the Getting Started example?
Currently, here is the way to add documents we show in the first Getting Started example:

```java
final String documents = "["
	+ "{\"book_id\": 123, \"title\": \"Pride and Prejudice\"},"
	+ "{\"book_id\": 456, \"title\": \"Le Petit Prince\"},"
	+ "{\"book_id\": 1, \"title\": \"Alice In Wonderland\"},"
	+ "{\"book_id\": 1344, \"title\": \"The Hobbit\"},"
	+ "{\"book_id\": 4, \"title\": \"Harry Potter and the Half-Blood Prince\"},"
	+ "{\"book_id\": 2, \"title\": \"The Hitchhiker\'s Guide to the Galaxy\"}"
	+ "]";
```

We would like to use, if it exists, a more friendly way to add documents that we can introduce into our Getting Started, instead of escaping double-quotes.

Maybe we could be inspired by the Stripe Java client and the `HashMap` solution, but it might also involve lots of lines, and this solution might be "heavy" too.
https://github.com/stripe/stripe-java#usage

What do you think?
Does someone know a way to make the Getting Started example more friendly? 😄

### PR DESCRIPTION
**Pull request by mohitsaxenaknoldus**
Fixes #67 

I think this looks cleaner and also removes the hassle of escaping quotes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,22 +70,28 @@ implementation 'com.meilisearch.sdk:meilisearch-java:0.4.1'
...
-      + "]";
-
+    JSONArray array = new JSONArray();
+    ArrayList items = new ArrayList() {{
+      add(new JSONObject().put("book_id", "123"));
+      add(new JSONObject().put("456", "Le Petit Prince"));
+      add(new JSONObject().put("1", "Alice In Wonderland"));
+      add(new JSONObject().put("1344", "The Hobbit"));
+      add(new JSONObject().put("4", "Harry Potter and the Half-Blood Prince"));
+      add(new JSONObject().put("2", "The Hitchhiker's Guide to the Galaxy"));
```
Comment (alallema) [inline README.md:90]:
```suggestion
      add(new JSONObject().put("book_id", "123").put("title", "Pride and Prejudice"));
      add(new JSONObject().put("book_id", "456").put("title", "Le Petit Prince"));
      add(new JSONObject().put("book_id", "1").put("title", "Alice In Wonderland"));
      add(new JSONObject().put("book_id", "1344").put("title", "The Hobbit"));
      add(new JSONObject().put("book_id", "4").put("title", "Harry Potter and the Half-Blood Prince"));
      add(new JSONObject().put("book_id", "2").put("title", "The Hitchhiker's Guide to the Galaxy"));
```
The structure of the documents was not good

**Review by alallema — COMMENTED** [review]
Thank you for this PR @mohitsaxenaknoldus!
I made a small change to make it work.

**Comment by mohitsaxenaknoldus** [comment]
> Thank you for this PR @mohitsaxenaknoldus!
> I made a small change to make it work.

Thank you so much! I've committed your suggestion.

**Review by alallema — APPROVED** [review]
LGTM!
Thank you for contributing to MeiliSearch. If you are participating in [Hacktoberfest](https://blog.meilisearch.com/contribute-hacktoberfest-2021/), and you would like to receive a small gift from MeiliSearch too, please complete [this form](https://meilisearch.typeform.com/to/rPexR5gq).

**Comment by alallema** [comment]
bors merge

**Pull request merged**